lib/binding/kvdb_interface.c

```diff
@@ -775,6 +776,84 @@ hse_kvs_delete(
     return err;
 }
 
+uint64_t
```

Comment:
Where did this come from? Needs to return an hse_err_t.

Reply:
Oh I see. This is a good change. I hated the experimental interface c file anyways.